src/site/content/en/progressive-web-apps/codelab-make-installable/index.md

```diff
@@ -54,6 +54,12 @@ To show the install prompt, call `prompt()` on the saved `beforeinstallprompt`
 event. Calling `prompt()` is done in the install button click handler because
 `prompt()` must be called from a user gesture.
 
+{% Aside 'gotcha' %}
+You need to open your app in its own browser tab. The side-by-side preview pane
+in Glitch doesn't work for this codelab, since it will be run in an iframe.
+Choose Glitch's "Preview in a new window" mode instead.
```

Comment:
I think we need more documentation than this, and, specifically, I claim that the codelab itself should be updated.

1. The "Preview in a new window" mode is only visible after you click the "Edit Fullscreen" button. (And if you click "Got It," the words "Edit Fullscreen" go away, and the button is an icon-only button with this icon. https://glitch.com/edit/edit/images/background-light/icons/fullscreen.svg It's likely impossible to find if you don't know what you're looking for or why you're looking for it.
2. Even when you do open in fullscreen mode, the "Preview in a new window" option is only visible after you click the "Preview" button.
3. It's so easy to make a mistake here, that I think the codelab itself should include code to detect whether it's being loaded in an iframe, and provide feedback to the user to explain the problem.

Reply:
One problem I see is that describing accurately how it looks on Glitch today may get outdated when they update their UI tomorrow.

Reply:
I claim the thing to do is to edit the code of the codelab, in the same style as the current requireHTTPS code. It should show a message like this:

```html
    <p id="requireTopLevel" class="hidden">
      <b>STOP!</b> This page <b>must not</b> be served in a frame.
      Please <a target="_top">click here to load the page as the top-level window</a>.
    </p>
```